tests/unittests/tests-color/tests-color.c

```diff
@@ -73,6 +73,18 @@ static void test_rgb2hex__success(void)
     TEST_ASSERT_EQUAL_INT(0x000AB13C, hex);
 }
 
+static void test_rgb2hsv__black(void)
+{
+    color_hsv_t hsv;
+    color_rgb_t rgb = { .r = 0x00, .g = 0x00, .b = 0x00 };
+
+    color_rgb2hsv(&rgb, &hsv);
+
+    TEST_ASSERT(0.0f == hsv.h);
+    TEST_ASSERT(0.0f == hsv.s);
+    TEST_ASSERT(0.0f == hsv.v);
```

Comment:
You should never compare floats for equality. Use the absolute value and a tolerance instead.

Reply:
Better?

Reply:
I'd use something like
```
   TEST_ASSERT(hsv.h < EPSILON);
```
where `EPSILON` is a "small enough" float constant.  What is small enough depends on the application and I don't remember the HSV code here, but maybe
```
   const float EPSILON = 1E-10f;
```

Reply:
Done.